code/modules/power/lighting.dm

```diff
@@ -149,13 +149,21 @@
 		if(LIGHT_STAGE_WIRED) icon_state = "bulb-construct-stage2"
 		if(LIGHT_STAGE_COMPLETE) icon_state = "bulb-empty"
 
+
 /obj/machinery/light_construct/spot
 	name = "large light fixture frame"
 	desc = "A large light fixture under construction."
-	icon = 'icons/obj/lighting.dmi'
+	icon = 'infinity/icons/obj/lighting_inf.dmi' // inf
+	icon_state = "spot-construct-stage1" // inf
```

Comment:
```suggestion
	//[INF]
	icon = 'infinity/icons/obj/lighting_inf.dmi'
	icon_state = "spot-construct-stage1"
	//[/INF]
```
https://github.com/infinitystation/Baystation12/blob/master/.github/CODE_GUIDELINES.md#%D1%80%D0%B5%D0%B4%D0%B0%D0%BA%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5-%D0%BE%D1%80%D0%B8%D0%B3%D0%B8%D0%BD%D0%B0%D0%BB%D1%8C%D0%BD%D0%BE%D0%B3%D0%BE-%D0%BA%D0%BE%D0%B4%D0%B0